controllers/clusterversion_controller.go

```diff
@@ -436,3 +465,44 @@ func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
 
 	return nil
 }
+
+func (c *ClusterVersionReconciler) reconcileCSIAddonsSubscription() error {
+	csiAddonsSubscription, err := c.getSubscriptionByPackageName(csiAddonsSubscriptionPackageName)
+	if err != nil {
+		return err
+
+	}
+
+	//the channel of csiAddons subscription here is stable-4.15 because of a direct merge to release-4.15
+	//this code shouldn't be back ported and from release-4.16 the logic is changed
+	csiAddonsSubscription.Spec.Channel = "stable-4.15"
+	if err := c.Update(c.ctx, csiAddonsSubscription); err != nil {
+		return fmt.Errorf("failed to update csi-addons subscription: %v", err)
+	}
+
+	if err != nil {
```

Comment:
what is this check for? seems redundant to me.

Reply:
removed